app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.tiles
+
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+import android.content.SharedPreferences
+import android.support.annotation.UiThread
+import org.json.JSONArray
+import org.json.JSONObject
+
+private const val HOME_TILES = "homeTiles"
```

Comment:
nit: Sorry I was unclear: I would have left the var name PREF (to indicate where it's used) but removed Pref from the value (because it's redundant with the usage). Also, I'd probably prefix with PREF since it makes it more obvious when multiple of these are used together that they're related.